pages/learn/develop/local-mode.md

```diff
@@ -104,21 +104,21 @@ balena push 192.168.1.37
 [Logs]    [1/25/2019, 1:14:21 PM] Installed service 'main sha256:97200548c13376aaf7445cb4c62fa13d2e758931cf34daf5ab20e3c00656a1b4'
 [Logs]    [1/25/2019, 1:14:21 PM] Starting service 'main sha256:97200548c13376aaf7445cb4c62fa13d2e758931cf34daf5ab20e3c00656a1b4'
 [Logs]    [1/25/2019, 1:14:22 PM] Started service 'main sha256:97200548c13376aaf7445cb4c62fa13d2e758931cf34daf5ab20e3c00656a1b4'
-[Logs]    [1/25/2019, 1:14:25 PM] [main] 
+[Logs]    [1/25/2019, 1:14:25 PM] [main]
 [Logs]    [1/25/2019, 1:14:25 PM] [main] > simple-server-node@1.0.0 start /usr/src/app
 [Logs]    [1/25/2019, 1:14:25 PM] [main] > node server.js
-[Logs]    [1/25/2019, 1:14:25 PM] [main] 
+[Logs]    [1/25/2019, 1:14:25 PM] [main]
 [Logs]    [1/25/2019, 1:14:28 PM] [main] Server listening on port  80
 ```
 
 These containers will have access to all the features and environment that balenaCloud deployed devices have. For example, you will still be able to query and use the [supervisor API][supervisor API] and the containers will be brought up automatically on boot.
 
 ## Caveats
-- In localMode, a device will not send logs back to the balenaCloud dashboard. 
+- In localMode, a device will not send logs back to the balenaCloud dashboard.
 - Set device and service environment variables from the dashboard will not be applied to localMode containers, but it is still possible to set these in your `docker-compose.yml` or `Dockerfile`.
 - Changes to Device configuration variables, for example `BALENA_HOST_CONFIG_gpu_mem`, will result in the device rebooting and applying those settings.
 - Actions such as `Restart` and `purge Data` from balenaCloud interface will not apply to localMode containers.
-- When switching out of localMode and back to tracking releases from balenaCloud, the balena supervisor will destroy any localMode containers and clean up unneeded base images, and then start up the application that balenaCloud instructs it to run.
+- When switching out of localMode and back to tracking releases from balenaCloud, the balena supervisor will destroy any localMode containers and  volumes as well as clean up unneeded base images, and then start up the application that balenaCloud instructs it to run.
```

Comment:
I guess it's also worth saying `images` too, else it sounds like only containers and volumes get cleaned up.

Reply:
Forgive my ignorance here, does the switch clean up all local images as well as "unneeded base images"?